### ISSUE TITLE
Documentation issue

### ISSUE THREAD

**Issue by johnsone006** [dotnet/dotnet-api-docs#9878]
I wish that when possible, the Microsoft learn website would always include examples of how objects from classes are used. For example, the AsycContentLoadedEventArgs class. The class page itself has a snippet that shows what class it inherits, but that's it. And  the method pages just include snippets that declare the method header with the parameters but not how those parameters are actually instantiated.

**Comment by carlossanlop** [comment]
Hello @johnsone006 - Do you have a link to the page you were looking for so we can take a look at the code example?

**Comment by johnsone006** [comment]
> Hello @johnsone006 - Do you have a link to the page you were looking for so we can take a look at the code example?

Yep. Here: https://learn.microsoft.com/en-us/dotnet/api/system.windows.automation.asynccontentloadedeventargs?view=windowsdesktop-8.0
Here: https://learn.microsoft.com/en-us/dotnet/api/system.windows.automation.asynccontentloadedeventargs.asynccontentloadedstate?view=windowsdesktop-8.0#system-windows-automation-asynccontentloadedeventargs-asynccontentloadedstate


And here: https://learn.microsoft.com/en-us/dotnet/api/system.windows.automation.asynccontentloadedeventargs.percentcomplete?view=windowsdesktop-8.0#system-windows-automation-asynccontentloadedeventargs-percentcomplete

**Comment by johnsone006** [comment]
Like to give an example of an event that's much better documented: https://learn.microsoft.com/en-us/dotnet/api/system.windows.automation.invokepattern.invokedevent?view=windowsdesktop-8.0#system-windows-automation-invokepattern-invokedevent
It would be nice to see how the async content loaded event would be subscribed to and what the event handler method might look like.
